docs/_includes/components/tooltips/tooltips-positions.html

```diff
@@ -0,0 +1,157 @@
+<!-- =================================================
+BEGIN: Tooltip Positions
+================================================== -->
+
+<section id="tooltips-positions">
+
+  <h2>
+
+    Tooltip Positions
+
+  </h2>
+
+  <p>
+
+    You can include more than body text in a message. Messages can support any HTML content. All type styling will be retained. However, if you add the class <code>.message-type-styling</code> to any typographic element, the preferred message type color and styling will be applied. For links, add the class <code>.message-link</code> to any <code>&lt;a&gt;</code>.
```

Comment:
Adding `message-link` class didn't seem to change anything for me. The link was always purple. Do we want to assume purple as the default color of links or white? I would lean towards white given that I think this is our preferred style for a link in this context (the purple contrast isn't great) and that in most cases developers will forget to add the additional class.